lib/engine/game/g_1822/game.rb

```diff
@@ -535,10 +533,6 @@ def all_potential_upgrades(tile, tile_manifest: false, selected_company: nil)
           upgrades = super
           return upgrades unless tile_manifest
 
-          upgrades |= [@green_s_tile] if self.class::UPGRADABLE_S_YELLOW_CITY_TILE == tile.name
```

Comment:
Looks like whole method  all_potential_upgrades could be removed?

Reply:
It's removed now